src/Model/ModelManager.php

```diff
@@ -666,4 +671,20 @@ private function getValueFromType($value, Type $type, string $fieldType, Abstrac
 
         return (string) $type->convertToDatabaseValue($value, $platform);
     }
+
+    private function getFieldCustomType(?string $fieldType): ?Type
```

Comment:
Even if this method is not part of the API, I think we should be consistent with the same principles we're enforcing in other packages about avoid returning `null` on getters when there is no value associated to the provided argument (see sonata-project/SonataAdminBundle#6050).